### PR TITLE
Revise list of actions in Chain module

### LINKED ISSUE #3230: Revise list of actions in the chain module
### Expected behavior
There must be unique and limited list of actions for the chain module. We must revisit all the actions and see how much we can optimize. 

### Actual behavior

Some actions got redundant in behaviour and some of which are not used. 

### Which version(s) does this affect? (Environment, OS, etc...)
1.7

### PR DESCRIPTION
**Pull request by limiaspasdaniel**
Cherry pick of: 89d1863747b655eb0f64789fbd38b2e8476fb62b

### What was the problem?
Unused actions where present in Chain codebase
### How did I fix it?
Removing unused actions
### How to test it?
Run all tests
### Review checklist

* The PR resolves #3230 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: framework/src/modules/chain/chain.js [hunk 1]
```diff
@@ -232,11 +232,6 @@ module.exports = class Chain {
...
 					action.params.signature
 				),
-			getLastConsensus: async () => this.scope.modules.peers.getLastConsensus(),
-			loaderLoaded: async () => this.scope.modules.loader.loaded(),
-			loaderSyncing: async () => this.scope.modules.loader.syncing(),
-			getForgersKeyPairs: async () =>
-				this.scope.modules.delegates.getForgersKeyPairs(),
 			getForgingStatusForAllDelegates: async () =>
 				this.scope.modules.delegates.getForgingStatusForAllDelegates(),
 			getForgersPublicKeys: async () => {
```
Comment (lsilvs) [inline framework/src/modules/chain/chain.js]: This is not being used and can be removed

File: framework/src/modules/chain/index.js [hunk 2]
```diff
@@ -61,10 +61,6 @@ module.exports = class ChainModule extends BaseModule {
 			getPeersCountByFilter: async action =>
 				this.chain.actions.getPeersCountByFilter(action),
 			postSignature: async action => this.chain.actions.postSignature(action),
-			getLastConsensus: async () => this.chain.actions.getLastConsensus(),
-			loaderLoaded: async () => this.chain.actions.loaderLoaded(),
-			loaderSyncing: async () => this.chain.actions.loaderSyncing(),
-			getForgersKeyPairs: async () => this.chain.actions.getForgersKeyPairs(),
 			getForgingStatusForAllDelegates: async () =>
 				this.chain.actions.getForgingStatusForAllDelegates(),
 			getForgersPublicKeys: async () =>
```
Comment (lsilvs) [inline framework/src/modules/chain/index.js]: Same as above

File: framework/src/modules/chain/chain.js (hunk 1 above)
Comment (limiaspasdaniel) [reply framework/src/modules/chain/chain.js]: Done

File: framework/src/modules/chain/index.js (hunk 2 above)
Comment (limiaspasdaniel) [reply framework/src/modules/chain/index.js]: Done

**Comment by limiaspasdaniel** [comment]
@shuse2 ready to merge

**Pull request merged**